tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java

```diff
@@ -74,6 +74,7 @@ public void testUnregister(ClusterInstance clusterInstance) {
 
     @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT})
     public void testClusterIdWithBootstrapController(ClusterInstance clusterInstance) {
+        System.out.println("bootstrapControllers: " + clusterInstance.bootstrapControllers());
```

Comment:
what is this?

Reply:
Sorry, this is my local test. Removed it. Thank you.